ot/mpz-ot-core/src/ferret/spcot/msgs.rs

```diff
@@ -0,0 +1,66 @@
+//! Messages for the SPCOT protocol
+
+use mpz_core::{hash::Hash, Block};
+use serde::{Deserialize, Serialize};
+
+/// A SPCOT message.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[allow(missing_docs)]
+pub enum Message<BaseMsg> {
+    BaseMsg(BaseMsg),
+    MaskBits(MaskBits),
+    ExtendFromSender(ExtendFromSender),
+    CotMsgForSender(CotMsgForSender),
+    CotMsgForReceiver(CotMsgForReceiver),
+    CheckFromReceiver(CheckFromReceiver),
+    CheckFromSender(CheckFromSender),
+}
```

Comment:
This protocol wraps a COT protocol, so we just add a variant for it. We can get rid of `BaseMsg` as that would be for a Base OT protocol which SPCOT doesn't use directly.

```suggestion
pub enum Message<CotMsg> {
    CotMsg(CotMsg),
    MaskBits(MaskBits),
    ExtendFromSender(ExtendFromSender),
    CheckFromReceiver(CheckFromReceiver),
    CheckFromSender(CheckFromSender),
}
```